index.html

```diff
@@ -5,8 +5,9 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta http-equiv="Content-Language" content="en">
-  <link rel="stylesheet" type="text/css" href="/css/github-syntax-highlight.css">
-  <link rel="stylesheet" type="text/css" href="/css/github-markdown.css">
+  <!-- theme stylesheets are dynamic loaded in index.js, according to 'theme' param -->
```

Comment:
Add `base.css` once you have extracted common rules.

Reply:
See https://github.com/instant-markdown/instant-markdown-d/pull/90#discussion_r884065632